packages/application-generic/src/services/queues/queue-base.service.ts

```diff
@@ -19,8 +19,18 @@ export class QueueBaseService {
     return this.instance;
   }
 
-  public createQueue(): void {
-    this.queue = this.instance.createQueue(this.topic, this.getQueueOptions());
+  public createQueue(overrideOptions?: QueueOptions): void {
+    const options = {
+      ...this.getQueueOptions(),
+      ...(overrideOptions && {
+        defaultJobOptions: {
+          ...this.getQueueOptions().defaultJobOptions,
+          ...overrideOptions.defaultJobOptions,
```

Comment:
Will this line will executed if overrideOptions is truthy?

Reply:
Yes, if `overrideOptions` is set it will be executed. Even if it is set as `{}` but with the spread we are safe nothing will override it.